lib/docker.ts

```diff
@@ -113,6 +138,7 @@ class Docker {
       statusCodes: {
         200: true,
         400: "bad request",
+        404: "not found",
```

Comment:
What is the piece of code that returns 404 now?

Reply:
https://github.com/snyk/snyk-docker-plugin/pull/222/files#diff-f17f8b9c3d9c0c016f5247574ebcee34R48

This call to `docker.save` could return a 404 if the container isn't already saved in the local docker daemon.